count.rb

```diff
@@ -0,0 +1,40 @@
+class WordCount
+  attr_accessor :speech
+  attr_reader :counted_words
+
+  def load_file(path)
+    @speech = File.read(path)
+  end
+
+  def parse_words
+    @speech.gsub!(/[^0-9a-zA-Z\s]/, '')
```

Comment:
If I load a valid text no substitution will be performed. Your return will be nil.